### ISSUE TITLE
failed to create process. {windows}

### ISSUE THREAD

**Issue by therealcodemonger** [Suyash458/soundcloud-dl#11]
Hi there, I am getting an error after following the steps -
i/e pip install soundcloud_dl

I have python 3.5

error : 

Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation.  All rights reserved.

C:\Users>sc-dl
failed to create process.

C:\Users>soundcloud-dl
failed to create process.

----------------------

Please help me out.

PS : I found out in some other sites that some added " " aound #!c:\program files (x86)\python35-32\python.exe but that didnt worked and gave me some other error.

Please rectify and help me out

**Comment by Mindstormer619** [comment]
Hi, the installation requires Python 2. It's mentioned on the README.

Closing this issue. Feel free to reopen if you think there's something else relevant to this.

**Comment by therealcodemonger** [comment]
so do i need to get python v2 as well.. hmmm or is it possible for you guys to make it compatible for 3+ as well :/ pls ?

**Comment by Mindstormer619** [comment]
I'm not sure if Python 3 support is on the cards. They're not really compatible with each other syntactically so it will basically need to be rewritten.

You're on Windows, right? You can get a second portable Python installation with Python 2.7 using WinPython or Anaconda. It won't mess with your existing Python installation.

**Comment by therealcodemonger** [comment]
Okay sure.. on it now, will update once I am on 2.x 


---Update---
@Mindstormer619: was able to get below and now it says url won't support: 


C:\Users\Rohit New\Anaconda2\Scripts>sc-dl
No URL entered.  // so no error this time

C:\Users\Rohit New\Anaconda2\Scripts>sc-dl help
usage: sc-dl [-h] [--url URL] [--dir DIR] [--all] [--likes]
             [--exclude EXCLUDE [EXCLUDE ...]]
             [--include INCLUDE [INCLUDE ...]] [--limit LIMIT]
             [--range RANGE RANGE]
sc-dl: error: unrecognized arguments: help

C:\Users\Rohit New\Anaconda2\Scripts>sc-dl "https://soundcloud.com/emotionalmusi
c/emotional-35"
usage: sc-dl [-h] [--url URL] [--dir DIR] [--all] [--likes]
             [--exclude EXCLUDE [EXCLUDE ...]]
             [--include INCLUDE [INCLUDE ...]] [--limit LIMIT]
             [--range RANGE RANGE]
sc-dl: error: unrecognized arguments: https://soundcloud.com/emotionalmusic/emot
ional-35

C:\Users\Rohit New\Anaconda2\Scripts>sc-dl https://soundcloud.com/emotionalmusic
/emotional-35
usage: sc-dl [-h] [--url URL] [--dir DIR] [--all] [--likes]
             [--exclude EXCLUDE [EXCLUDE ...]]
             [--include INCLUDE [INCLUDE ...]] [--limit LIMIT]
             [--range RANGE RANGE]
sc-dl: error: unrecognized arguments: https://soundcloud.com/emotionalmusic/emot
ional-35

/// tried with n w/o quotes-----no go 

C:\Users\Rohit New\Anaconda2\Scripts>

----

Not sure where I am going wrong, meanwhile I was able to use SC's chrome extension which worked smoothly but still would like to use this script at least once :/

**Comment by therealcodemonger** [comment]
I downloaded SoundScrape  finally and it did the job :) https://github.com/grohit19/SoundScrape

**Comment by suyashb95** [comment]
Hi, you have to pass arguments to sc-dl with keywords. So the correct way to do it would be 
```
sc-dl --url  "https://soundcloud.com/emotionalmusic/emotional-35"
```
I guess I'll remove the keyword for the URL parameter alone but the rest will have to stay.
SoundScrape beat me to the 'One downloader to rule them all' project hahah

**Comment by therealcodemonger** [comment]
Yeah sc-dl --URL <URL> did the job but it's working when I run the cmd promt only in the folder where the sc-dl script is present. I added it to the environment path but no go. 

And yeah soundscrape seems ruling :p 

The soundscrape cmd works in all drives / folders / locations but even this script gave me error. 

I had added " after #! And enclosed again with " - the path address of Python , which seems to be a bug with pip which I got to know from stacksoverflow. 

So all in all I'm good with SS :D but will look forward for your script in future as well :)

Cheers !!
